File: Demo/FrmDemo1.vb
Imports Demo.Z80NavBar
Imports Demo.Z80NavBar.Themes

Public Class FrmDemo1
    Public Sub New()
        InitializeComponent()
        Z80_Navigation1.Initialize(New DemoItems().sample1, New ThemeSelector(Theme.Dark).CurrentTheme)
    End Sub

    Private Sub Z80_Navigation1_SelectedItem(item As NavBarItem) Handles Z80_Navigation1.SelectedItem
        LblInfo.Text = $"CONTENT SAMPLE -> ID: {item.ID} Text: {item.Text}"
    End Sub

    Private distanceCopy As Integer
    Private Sub BtnShowHide_Click(sender As Object, e As EventArgs) Handles BtnShowHide.Click
        If (SplitContainer1.SplitterDistance > 37) Then
            distanceCopy = SplitContainer1.SplitterDistance
            SplitContainer1.SplitterDistance = 37
        Else
            SplitContainer1.SplitterDistance = distanceCopy
        End If
    End Sub
End Class

File: DemoCS/FrmDemo1.cs
using System;
using System.Windows.Forms;
using Z80NavBar;
using Z80NavBar.Themes;

namespace DemoCS
{
    public partial class FrmDemo1 : Form
    {
        public FrmDemo1()
        {
            InitializeComponent();
            z80_Navigation1.SelectedItem += Z80_Navigation1_SelectedItem;
            z80_Navigation1.Initialize(new DemoItems().sample1, new ThemeSelector(Theme.Dark).CurrentTheme);
        }

        private void Z80_Navigation1_SelectedItem(NavBarItem item)
        {
            LblInfo.Text = $"CONTENT SAMPLE -> ID: {item.ID} Text: {item.Text}";
        }

        private int distanceCopy;
        private void BtnShowHide_Click(object sender, EventArgs e)
        {
            if (splitContainer1.SplitterDistance > 37)
            {
                distanceCopy = splitContainer1.SplitterDistance;
                splitContainer1.SplitterDistance = 37;
            }
            else
                splitContainer1.SplitterDistance = distanceCopy;
        }
    }
}
